src/tools/dev/scripts/bv_support/bv_mxml.sh

```diff
@@ -23,7 +23,7 @@ function bv_mxml_info
     export MXML_FILE=${MXML_FILE:-"mxml-2.6.tar.gz"}
     export MXML_VERSION=${MXML_VERSION:-"2.6"}
     export MXML_COMPATIBILITY_VERSION=${MXML_COMPATIBILITY_VERSION:-"2.6"}
-    export MXML_BUILD_DIR=${MXML_BUILD_DIR:-"mxml-2.6"}
+    export MXML_SRC_DIR=${MXML_SRC_DIR:-"mxml-2.6"}
```

Comment:
This module has been removed on develop.

Reply:
Removed as well.